src/main/java/org/folio/search/configuration/properties/FolioKafkaProperties.java

```diff
@@ -26,6 +28,16 @@
    */
   private long retryDeliveryAttempts;
 
+  /**
+   * List of default topics to maintain backward compatibility.
+   */
+  private String topicsRegexp;
```

Comment:
The name `topicsRegexp` is missleading, as you also have `topicPattern`. If it is legacy, let's name it `legacyTopicPattern` or `defaultTopicPattern` and mark as @Deprecated

Reply:
Yes, thanks for noticing this, this change is not required at all